index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify the <a>holder</a> to the verifier.
+<a>verifier</a>.
```

Comment:
Typo got in here (probably from suggestion mode somewhere):

```suggestion
to identify the <a>holder</a> to the <a>verifier</a>.
```